### PR TITLE
NFT Items in Unreal with Thirdweb

### PR DESCRIPTION
**Pull request by briandoyle81CB**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md [hunk 1]
```diff
@@ -0,0 +1,773 @@
...
+
+By the end of this tutorial you should be able to:
+
+- Pull a user's NFTs into an Unreal Engine Game
+- Apply elements from the NFT to game entities
+- Award NFTs to players for game accomplishments or actions
+
+---
+
+## Prerequisites
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md:31]: Maybe add HDD requirements + XCode?

I deleted Xcode to free up space and realized I needed xcode to run Unreal Engine.

Comment (hughescoin) [reply apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md:31]: Add Postgres to prereqs

Comment (briandoyle81CB) [reply apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md:31]: Xcode is covered under the Unreal Engine one, they have you install it in the setup and we're assuming that is already done.

For postgres, you just need to install it, which is inline below.  You don't need to know anything about it or do anything with it for this tutorial.

File: apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md [hunk 2]
```diff
@@ -0,0 +1,773 @@
...
+Finally, expose this function to the editor.
+
+```c++
+// ThirdwebManager.h
+// Function to perform the get NFT Colors operation
+UFUNCTION(BlueprintCallable, Category = "Thirdweb")
+void GetNFTColors();
+```
+
+Compile and reload the project in the editor.
```
Comment (briandoyle81CB) [inline apps/base-docs/tutorials/docs/1_thirdweb-unreal-nft-items.md:537]: I'm not sure why github is rendering this as blue, but it looks normal in preview in code and running locally

**Review by hughescoin — APPROVED** [review]

**Pull request merged**